packages/office-addin-lint/src/defaults.ts

```diff
@@ -3,11 +3,11 @@
 
 import * as usageData from "office-addin-usage-data";
 
-export const lintFiles = "src/**/*.{ts,tsx,js,jsx}";
+export const lintFiles = "src\\**\\*.{ts,tsx,js,jsx}";
```

Comment:
Should keep forward slash. This should work on Windows too.

Reply:
Will change in latest commit now that forward slash is working on my windows machine.